src/content/posts/accouncing-juniortrack.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "🎉 Announcing Junior Track 🎉"
+date: 
+draft: false
+---
+
+**Have you ever wanted to go to a conference but...**
+
+- weren't sure if it was right for you?
+- it was too expensive (usually conference tickets are $300+)
+- weren't sure whether you'd fit in?
+
+Enter DDD. 
+
+This year, DDD has expanded to include development, design, data, and a fourth track focused on getting juniors involved in the community, and inspired beyond the norm of their usual day-to-day experience.
+
+Buy your early bird tickets now - only $50: http://next.dddsydney.com.au/tickets/
+
+**Benefits of attending DDD Sydney:**
+
+- discover something new in the dev world
+- meet like-minded individuals, including other juniors!
+- explore dev, design and data
+- learn skills to further your career
+Find out more at http://next.dddsydney.com.au/
+
+**What to expect**
+
+- Fully catered
+- Inclusive atmosphere
+- Interesting presentations*
+- Awesome people
+
+*In terms of talks, that's up to you! Unlike most conferences, DDD leaves talk selection up to you, the attendees.
+
+We want as diverse a pool as possible in terms of talk ideas (_more info below_). Once submissions are in,the public will vote on your anonymous submissions to decide what we hear on the day based on the title and abstract (only). This is your conference, you're in control. Make sure you vote!
+
+**Have you ever wanted to give a talk?**
+
+DDD Sydney is looking for speakers just like you!
+
+We want to create an inclusive and comfortable environment for our speakers and attendees. To make speakers feel a little more comfortable:
+
+- Submissions are open to everyone and are language and technology agnostic.
+- Talks can be solo or multi-presenter talks - whichever you're more comfortable with.
+- We also have a mentoring service available if you need help with your submission or preparing the talk itself.
+- Talks are voted on anonymously by the community. No one will know you've submitted a talk unless you want to tell people. You're safe to fail here 🙂
+
+**Not sure what to speak about?**
+
+From Front-end, Back-end, Dev-Ops, Design, Data, and beyond - we are looking for talks to inspire. From technical talks to the more personal - here are some suggestions:
+
+- Front-End
+- Back-End
+- Dev-ops
+- Creative Coding
+- IoT
+- Mobile
+- Career development
+- Your journey
+- UX, UI, Design
+- Data
+
+Don't forget, we're holding a free CFP workshop Sunday 20 May to help you brainstorm and refine your submission. RSVP [here] (https://www.eventbrite.com.au/e/ddd-sydney-call-for-papers-workshop-tickets-45190507023)
```

Comment:
Need to strip the space between the `]` and `(` so Markdown will turn it into a proper link.